src/compute/fixed_msm_tests.rs

```diff
@@ -0,0 +1,60 @@
+use super::*;
+
+use crate::compute::ElementP2;
+use ark_bls12_381::G1Affine;
+use ark_std::UniformRand;
+use curve25519_dalek::ristretto::RistrettoPoint;
+use rand_core::OsRng;
+
+#[test]
+fn we_can_compute_msms_using_a_single_generator() {
+    let mut rng = OsRng;
```

Comment:
I had to add `init_backend();` to the top of each of these tests in order for them to run.